lib/src/main/java/com/basistheory/android/model/CreateSessionResponse.kt

```diff
@@ -0,0 +1,19 @@
+package com.basistheory.android.model
+
+import com.google.gson.annotations.SerializedName
+import java.time.OffsetDateTime
+
+class CreateSessionResponse(
+    @SerializedName("session_key") var sessionKey: String,
```

Comment:
Do we really need the gson serialization annotations on these models? They should never be serialized by us (we only map to/from the java sdk which is what handles de/serialization, so we have no requirement to keep these and it might lead to consumers depending on something extraneous that we want to delete or remove later.